### PR TITLE
replaced all main menu images

### PR DESCRIPTION
**Pull request by JaredTheHammer**
Hi Gavin, pretty basic swapping of pictures for main menu but I'm practicing doing the pull request process.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@pixi/filter-glow": "2",
+    "@pixi/filter-glow": "^2.7.0",
```
Comment (GavPierce) [inline client/package.json:11]: I downgraded pixie originally to have a version that worked with our little older node project. Are we sure 2.7 is working fine? (No issues starting a game)

**Review by GavPierce — APPROVED** [review]
Looks good! Just one comment. But if that is not an issue you're good to merge.

File: client/package.json (hunk 1 above)
Comment (JaredTheHammer) [reply client/package.json:11]: I had no issues starting a game. I just installed it and this was the default. I didn't actively change the version.

**Comment by GavPierce** [comment]
@JaredTheHammer great! You're ready to merge then

**Pull request merged**